### PR TITLE
Cleanup cron trigger type from contract

### PR DESCRIPTION
**Pull request by infiloop2**

### CODE REVIEW

**Review by RyanRHall — DISMISSED** [review]

File: contracts/pnpm-lock.yaml
Comment (FelixFan1992) [inline contracts/pnpm-lock.yaml:1]: do we need this change?

File: contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol [hunk 1]
```diff
@@ -248,7 +247,7 @@ abstract contract KeeperRegistryBase2_1 is ConfirmedOwner, ExecutionPrevention {
    * @member balance the balance of this upkeep
    * @member admin for this upkeep
    * @member maxValidBlocknumber until which block this upkeep is valid
-   * @member lastPerformedBlockNumberOrTimestamp the last block number or timestamp when this upkeep was performed
+   * @member lastPerformedBlockNumber the last block number or timestamp when this upkeep was performed
```
Comment (FelixFan1992) [inline contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol:250]: update the comment

File: contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol [hunk 2]
```diff
@@ -275,7 +274,7 @@ abstract contract KeeperRegistryBase2_1 is ConfirmedOwner, ExecutionPrevention {
    * @member paused if this upkeep has been paused
    * @member amountSpent the amount this upkeep has spent
    * @member balance the balance of this upkeep
-   * @member lastPerformedBlockNumberOrTimestamp the last block number or timestamp when this upkeep was performed
+   * @member lastPerformedBlockNumber the last block number or timestamp when this upkeep was performed
```
Comment (FelixFan1992) [inline contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol:277]: same

File: contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol (hunk 1 above)
Comment (infiloop2) [reply contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol:250]: will do in a followup, takes a while for this merge and I have dependencies on top of it

File: contracts/pnpm-lock.yaml
Comment (infiloop2) [reply contracts/pnpm-lock.yaml:1]: nice catch, i think this might be due to different version of pnpm locally. Will redo

**Pull request merged**